modules/tibc/apps/nft_transfer/client/cli/flags.go

```diff
@@ -0,0 +1,20 @@
+package cli
+
+import (
+	flag "github.com/spf13/pflag"
+)
+
+const (
+	FlagRelayChain = "relayChain"
+	FlagAwayFromChain = "awayFromChain"
+)
+
+var (
+	FsNftTransfer    = flag.NewFlagSet("", flag.ContinueOnError)
+)
+
+func init() {
+	FsNftTransfer.String(FlagRelayChain, "", "relay chain used by cross-chain NFT")
+	FsNftTransfer.String(FlagAwayFromChain, "", "indicates whether nft is far away from the source")
```

Comment:
There is no need to declare whether it is IsAwayFromChain or false, we can decide through the NFT class, source_chain, and dest_chain in Keeper.